src/sonic-yang-models/yang-models/sonic-qos-pfc.yang

```diff
@@ -0,0 +1,82 @@
+module sonic-qos-pfc {
+    namespace "http://github.com/Azure/sonic-qos-pfc";
+    prefix sonic-qos-pfc;
+
+    yang-version 1.1;
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC QoS Priority-based Flow Control";
+
+    revision 2021-07-01 {
+        description
+            "Initial revision.";
+    }
+
+    container sonic-qos-pfc {
+
+        container PFC_WD {
+            list PFC_WD_LIST {
+                key "ifname";
+
+                leaf ifname {
+                    mandatory true;
+                    type string {
+                        pattern "Ethernet([1-3][0-9]{3}|[1-9][0-9]{2}|[1-9][0-9]|[0-9])|GLOBAL" {
+                            error-message "Invalid interface name";
+                            error-app-tag interface-name-invalid;
+                        }
+                    }
+                }
+
+                leaf action {
+                    default drop;
+                    type enumeration {
+                        enum drop;
+                        enum forward;
+                        enum  alert;
+                    }
+                    description
+                        "PFC watchdog action when entering storm state.";
+                }
+
+                leaf detection_time {
+                    default 200;
+                    type uint32 {
+                        range 100..5000;
+                    }
+                    description
+                        "Detection interval for pause storm.";
+                }
+
+                leaf restoration_time {
+                    default 200;
+                    type uint32 {
+                        range 100..60000;
+                    }
+                    description
+                        "Time delay before resuming normal PFC operation.";
+                }
+
+                leaf POLL_INTERVAL {
```

Comment:
Leaf name should not be in the upper case.

Reply:
POLL_INTERVAL is in UpperCase as this attribute is in Uppercase in Config DB.
Here is the sample-
        "PFC_WD": {
            "Ethernet9": {
               "action": "drop",
               "detection_time": "100",
               "restoration_time": "400"
            }
        },
        "PFC_WD": {
            "GLOBAL": {
                "POLL_INTERVAL": "100"
            }
        },